### ISSUE TITLE
[BUG] Auto confirm does not auto confirm your garbage tote swap decision

### ISSUE THREAD

**Issue by Fawoosh** [loathers/garbage-collector#1873]
the popup window that asks if you are okay with discarding your shirt scraps or champagne leftovers does not get confirmed via auto confirm nor does it time out and default to a choice.

**Comment by horrible-little-slime** [comment]
Unfortunately, that popup comes from mafia rather than garbo, so we aren't able to handle it in the scope of autoconfirm. I think @Malibu-Stacey was aware of a workaround to not get mafia to give this popup? and I have no recollection of why we haven't implemented that, but I'm pretty sure there was a reason?

**Comment by Fawoosh** [comment]
:(

**Comment by eegeeZA** [comment]
![image](https://github.com/loathers/garbage-collector/assets/2684652/13455b21-301f-421c-bb5b-c3ab6d885488)

I think it's triggered by this: https://github.com/loathers/garbage-collector/blob/632c5c084cc4a762a0e2188d8b97efdf9831452b/packages/garbo/src/fights.ts#L2044

**Comment by Fawoosh** [comment]
> ![image](https://private-user-images.githubusercontent.com/2684652/329324537-13455b21-301f-421c-bb5b-c3ab6d885488.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUyODE4OTMsIm5iZiI6MTcxNTI4MTU5MywicGF0aCI6Ii8yNjg0NjUyLzMyOTMyNDUzNy0xMzQ1NWIyMS0zMDFmLTQyMWMtYmI1Yi1jM2FiNmQ4ODU0ODgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUwOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MDlUMTkwNjMzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YzU0NjQ5MmFjZjljNDc5YzgzZmNjNGQyMGFiNzhjZThhNDdjYTBkNGI5YjljZGFmN2M4MmZjNjA2ZmFiN2E5MCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.ZRTJ629IdwrImwMCBYkhzPx4imoHip6pZyPooMQMnck)
> 
> I think it's triggered by this:
> 
> https://github.com/loathers/garbage-collector/blob/632c5c084cc4a762a0e2188d8b97efdf9831452b/packages/garbo/src/fights.ts#L2044

Yes, I am aware of what its triggered by. The issue is it does not get auto confirmed which neil informed me is a mafia thing.

**Comment by eegeeZA** [comment]
Cool, for now I'll short circuit the code to avoid folding.
![image](https://github.com/loathers/garbage-collector/assets/2684652/a2b51154-112f-4c49-857a-b92b592bfc29)

**Comment by frazazel** [comment]
This could be solved by setting `_garbageItemChanged` to true, to lie to mafia.
